SPECS/libmicrohttpd/libmicrohttpd.spec

```
@@ -1,21 +1,29 @@
-Summary:        Lightweight library for embedding a webserver in applications
 Name:           libmicrohttpd
-Version:        0.9.76
-Release:        1%{?dist}
-License:        LGPLv2+
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
-URL:            https://www.gnu.org/software/libmicrohttpd/
+Distribution:   Microsoft Azure Linux
```

Let's use `Azure Linux` instead. There's [a bulk update](https://github.com/microsoft/CBL-Mariner/pull/7606/) underway to modify all specs.
```suggestion
Distribution:   Azure Linux
```

---

ack